migrations/20191125211848_create_roles_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE roles (
+	id integer PRIMARY KEY,
+	role_type text
+);
+
+CREATE TABLE user_role (
```

Comment:
this should be pluralized 
```suggestion
CREATE TABLE user_roles (
```